### PR TITLE
Feature/vehicle management

### PR DESCRIPTION
**Pull request by gabrigio30**
Added vehicle_model admin and generic tests

### CODE REVIEW

File: .DS_Store
Comment (lovrojakic) [inline .DS_Store:1]: Can we add .DS_Store to .gitignore?

Comment (aliazani) [reply .DS_Store:1]: Yes, feel free to add it.

File: vehicle/tests/models/vehicle_model_tests.py
Comment (aliazani) [inline vehicle/tests/models/vehicle_model_tests.py:1]: For being consistent pls use BDD style.

**Comment by aliazani** [comment]
Test ordering didn't pass.
@gabrigio30

**Pull request merged**